scripts/apps/monitoring/directives/MonitoringView.js

```diff
@@ -41,31 +40,45 @@ export function MonitoringView($rootScope, authoringWorkspace, pageTitle, $timeo
             };
 
             var updateTimeout;
+            var userHasScrolled = false;
 
             function handleContainerScroll($event) {
-                if ($rootScope.itemToogle) {
-                    scope.$applyAsync(() => {
-                        $rootScope.itemToogle(false);
-                        $rootScope.itemToogle = null;
-                    });
+                // perform only when user has scrolled
+                if (userHasScrolled) {
+                    if ($rootScope.itemToogle) {
+                        scope.$applyAsync(() => {
+                            $rootScope.itemToogle(false);
+                            $rootScope.itemToogle = null;
```

Comment:
This was an existing code, seems intend to close ActionMenu(from preview pane) when its open and user scroll, so i keep that and just corrected the spelling.